lib/chef/knife/user_list.rb

```diff
@@ -34,8 +34,35 @@ class UserList < Knife
         :long => "--with-uri",
         :description => "Show corresponding URIs"
 
+      def osc_11_warning
+<<-EOF
+knife user list failed in a way that indicates that you are using an Open Source 11 Server.
+knife user list for Open Source 11 Server is being deprecated.
+Open Source 11 Server user commands now live under the knife opc_user namespace.
```

Comment:
Yeah, in general I'm not down with the copy-pasta and I would have used a smarter solution, but this code is going away in a few months when 11 support stops, so I didn't wanna put too much effort into the parts that were going away. Hard to balance but the stuff that is sticking around I tried to make as high-quality as possible (with many many tests).